deploy/kiali/kiali_cr.yaml

```diff
@@ -492,22 +495,37 @@ spec:
 #      url: ""
 #
 # **Istio-specific settings:
+# component_status:
+#   enabled: Enable/Disable of istio component status into masthead indicator. It defaults to true.
+#   components: A list of components that Kiali will check its statuses.
+#     app_label: Istio component pod app label.
+#     is_core: Whether the component is core for your deployment.
 # istio_identity_domain: The annotation used by Istio to identify domains.
 # istio_sidecar_annotation: The pod annotation used by Istio to identify the sidecar.
-# istio_status_enabled: Enable/Disable of istio component status into masthead indicator. It defaults to true.
 # url_service_version: The Istio service used to determine the Istio version. If empty, assumes the URL for the well-known Istio version endpoint.
 #    ---
 #    istio:
+#      component_status:
+#        enabled: true
+#        components:
+#          - app_label: istiod
+#            is_core: true
+#          - app_label: istio-ingressgateway
+#            is_core: true
+#          - app_label: istio-egressgateway
+#            is_core: false
```

Comment:
Changes here just to remain consistent with how it is really defined in the defaults/main.yml (the order is reversed):
```suggestion
#          - app_label: istio-egressgateway
#            is_core: false
#          - app_label: istio-ingressgateway
#            is_core: true
#          - app_label: istiod
#            is_core: true
```

Reply:
I'd prefer to change the defaults to be exact as in here. I'd rather go for this order.

Reply:
That's fine. Just so the order in the list is the same for consistency.